src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -8,14 +8,22 @@
 @import '../../styles/shared-styles.scss';
 @import '../../../node_modules/office-ui-fabric-core/dist/sass/References';
 
-$font-size: var(--font-size, #{$ms-font-size-m});
-$font-weight: var(--font-weight, #{$ms-font-weight-semibold});
-$color: var(--color, #{$ms-color-neutralDark});
+$person-card-display-name-font-size: var(--person-card-display-name-font-size, 27.3103px);
+$person-card-title-font-size: var(--person-card-title-font-size, 18.2069px);
+$person-card-subtitle-font-size: var(--person-card-subtitle-font-size, 15.1724px);
+$person-card-details-title-font-size: var(--person-card-details-title-font-size, 14px);
+$person-card-details-item-font-size: var(--person-card-additional-details-item-font-size, 15px);
+$person-card-display-name-color: var(--person-card-display-name-color, #333333);
+$person-card-title-color: var(--person-card-title-color, #767676);
+$person-card-subtitle-font-color: var(--person-card-subtitle-font-color, #767676);
+$person-card-details-title-color: var(--person-card-details-title-color, #000000);
+$person-card-additional-details-item-color: var(--person-card-additional-details-item-color, #201f1e);
```

Comment:
Good point, didn't think about the hyperlinks. We should probably add additional properties for hyperlinks in general in the component - but I propose we add that as a feature after 1.2. What do you think?

Reply:
My thoughts: 
I decided to leave out the possibility of styling hyperlinks since they are very susceptible  to accessibility issues. AFAIK, the main css properties available are, color, outline, text-decoration(several properties), but then we should also expose separate variables for `:active` `:hover` etc... To me this seems like will be a common issue across any component with hyperlinks, so maybe this is easier to handle in a 1.2 as a subcomponent?

Reply:
If you mean 2.0, then we are in agreement :)

Reply:
updated property to `--person-card-details-item-color`:
5389a0c